src/main/java/org/broadinstitute/hellbender/tools/recalibration/RecalibrationArgumentCollection.java

```diff
@@ -41,25 +43,13 @@
     public File RECAL_TABLE_FILE = null;
     public PrintStream RECAL_TABLE;
 
-    /**
-     * Note that the --list argument requires a fully resolved and correct command-line to work.
-     */
-    @Argument(fullName = "list", shortName = "ls", doc = "List the available covariates and exit", optional = true)
-    public boolean LIST_ONLY = false;
 
-    /**
-     * Note that the ReadGroup and QualityScore covariates are required and do not need to be specified.
-     * Also, unless --no_standard_covs is specified, the Cycle and Context covariates are standard and are included by default.
-     * Use the --list argument to see the available covariates.
-     */
-    @Argument(fullName = "covariate", shortName = "cov", doc = "One or more covariates to be used in the recalibration. Can be specified multiple times", optional = true)
-    public List<String> COVARIATES = new ArrayList<>();
+    //HACK - we hardwire those names here to keep compatibility with GATK3 reports. However, using a report with a different set will cause a UserException.
+    @Deprecated
+    public List<String> COVARIATES = Covariate.classNameList(new StandardCovariateList());
 
-    /*
-     * The Cycle and Context covariates are standard and are included by default unless this argument is provided.
-     * Note that the ReadGroup and QualityScore covariates are required and cannot be excluded.
-     */
-    @Argument(fullName = "no_standard_covs", shortName = "noStandard", doc = "Do not use the standard set of covariates, but rather just the ones listed using the -cov argument", optional = true)
+    //HACK - we hardwire those names here to keep compatibility with GATK3 reports. However, using a report with a different set will cause a UserException.
+    @Deprecated
     public boolean DO_NOT_USE_STANDARD_COVARIATES = false;
```

Comment:
Was there really no way to get rid of these? They are not even annotated as arguments anymore, yet are still in the `RecalibrationArgumentCollection`. At the very least make them final, and wrap `COVARIATES` in a `Collections.unmodifiableList()`.